File: java-poc/src/main/java/io/craigmiller160/querydslpoc/java/data/repository/PositionRepository.java
package io.craigmiller160.querydslpoc.java.data.repository;

import io.craigmiller160.querydslpoc.java.data.entity.Position;
import java.util.UUID;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PositionRepository extends JpaRepository<Position, UUID> {}

File: kotlin-poc/src/main/kotlin/io/craigmiller160/querydslpoc/kotlin/data/repository/PositionRepository.kt
package io.craigmiller160.querydslpoc.kotlin.data.repository

import io.craigmiller160.querydslpoc.kotlin.data.entity.Position
import java.util.UUID
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository interface PositionRepository : JpaRepository<Position, UUID>
